src/app/public/modules/popover/popover.component.ts

```diff
@@ -173,11 +180,14 @@ export class SkyPopoverComponent implements OnInit, OnDestroy {
     private adapterService: SkyPopoverAdapterService,
     private changeDetector: ChangeDetectorRef,
     private elementRef: ElementRef,
-    private windowRef: SkyWindowRefService
+    private windowRef: SkyAppWindowRef
   ) { }
 
   public ngOnInit(): void {
     this.preferredPlacement = this.placement;
+  }
+
+  public ngAfterViewInit(): void {
```

Comment:
Why did you move this to `AfterViewInit`?

Reply:
Angular 9 updated `ViewChild` to require we set a `static` property. The default value is `static: false`, which means that the element is only accessible in the `ngAfterViewInit`. Angular had some mechanisms in place before version 9 that would allow for accessing children in `ngOnInit`, but now we'd have to declare the child `static: true`, which I'm uncomfortable doing since it would be new behavior. I'm open to changing it in a future pull request but for now I wanted to maintain existing functionality. Good eye!

Reply:
Ok, thanks for the explanation. Considering the alternatives, I think what you did here is probably the safest route. 👍